File: public/phase-2-20160620022203/C/a-bag.c
// Name : A Bag
#include <stdio.h>
int main() {
	// input:
	// n : number of toolkits
	// volume_of_each_toolkit : volume of each toolkit
	int n, volume_of_each_toolkit[10000], minimum_number_of_bags=4, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &volume_of_each_toolkit[index]);
	}


	// write your code here
	// store your results in `minimum_number_of_bags`

	// output
	printf("%d", minimum_number_of_bags);
	return 0;
}

File: public/phase-2-20160620022203/JAVA/a-bag.java
// Name : A Bag
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : number of toolkits
		// volume_of_each_toolkit : volume of each toolkit
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] volume_of_each_toolkit = new int[n];
		String[] volume_of_each_toolkit_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			volume_of_each_toolkit[index] = Integer.parseInt(volume_of_each_toolkit_elements[index]);


		// write your code here
		// store your results in `minimum_number_of_bags`

		// output
		// Dummy Data
		int minimum_number_of_bags=4;

		System.out.println(minimum_number_of_bags);

	}
}